### PR TITLE
Fixes for legacy pages

### PR DESCRIPTION
**Pull request by adborden**
Depends on #288 

Fixes for #180 some of #179

### CODE REVIEW

**Comment by adborden** [comment]
@hbillings this is ready for review :)

File: www.foia.gov/_includes/legacy/popup.html [hunk 1]
```diff
@@ -11,3 +11,23 @@
        </div>
 </div>
 <!-- End Popup Window -->
+<!-- Start Popup Window -->
```
Comment (hbillings) [inline www.foia.gov/_includes/legacy/popup.html]: Should we add a note about which pages use one popup vs. the other? The video link is clearly its own thing, but these two are confusing together.

File: www.foia.gov/_sass/theme/_temp_fixes.scss [hunk 2]
```diff
@@ -31,13 +31,17 @@ ul.tabs li {
...
 /* this is necessary because the markup in the HTML doesn't match the markup WDS expects, so it hides the whole thing */
-#selectFY [type=checkbox],
-#selectFY [type=radio] {
-  position: static;
-  left: 0;
-  display: inline-block;
-  width: 10px;
-  height: 1em;
+// TODO https://github.com/18F/beta.foia.gov/issues/178 
+#componentSelectionData, // This markup comes from the Tomcat backend, it is **not** a temporary fix.
```
Comment (hbillings) [inline www.foia.gov/_sass/theme/_temp_fixes.scss]: I'm wondering if we should rename this file to `legacy_overrides` or something -- the `temp fix` language isn't the best for where we are now.

**Review by hbillings — APPROVED** [review]
⚡️

**Pull request merged**